cmd/default-domain/main.go

```diff
@@ -153,13 +200,20 @@ func main() {
 		return
 	}
 
-	// Look up the address for IngressGateway.
-	address, err := waitForIngressGatewayAddress(kubeClient)
+	// Start an HTTP Server
+	h := network.NewProbeHandler(http.HandlerFunc(func(w http.ResponseWriter, _ *http.Request) {
```

Comment:
This is just new for health check, correct?

Reply:
This is to correctly respond to the network probe for ingress readiness